source/deployment/bulk-loading.rst

```diff
@@ -0,0 +1,27 @@
+.. _bulk-loading:
+
+=================
+Bulk Loading Data
+=================
+
+Large quantities of data can be imported from a JSON file into Mattermost at the command line using the bulk loading feature. This feature is most suitable for migrating data from an existing messaging system, or for pre-populating a new installation with data.
```

Comment:
I think it's better as-is. When you say "using the bulk loading feature from the command line" you're implying that you can use it from somewhere else; a GUI for example. In this case, the command line is the only way to use the bulk loading feature.